tests/python/relay/test_ir_parser.py

```diff
@@ -0,0 +1,530 @@
+from tvm.relay.parser import enabled
+if not enabled():
```

Comment:
we should not do the exit here, this will likely cause the problem of test. Instead, in each test function, check relay.parser.enabled() and return